src/action.c

```diff
@@ -688,40 +688,41 @@ show_menu(struct server *server, struct view *view, struct cursor_context *ctx,
 	if (pos_x && pos_y) {
 		struct output *output = output_nearest_to(server,
 				server->seat.cursor->x, server->seat.cursor->y);
+		struct wlr_box usable = output_usable_area_in_layout_coords(output);
```

Comment:
Pre-existing issue but couldn't `output_nearest_to()` return NULL (e.g. if there is no output connected at all)?